data/ondemandflightstatus.md

```diff
@@ -0,0 +1,186 @@
+## On Demand Flight Status
+
+In test environment, the On Demand Flight Status API contains cached data.
```

Comment:
Just to be precise, On-Demand Flight Status data are not cached.

There are 2 sources of data to build the API response:
- Amadeus schedule for flight information (origin, destination, scheduled date and time, etc...)
- OAG/other feed for the update on the scheduled/planned flight information (delay, terminal assignment at the airport, etc...)

In test, the Amadeus schedule information is a copy of the production at a given time that now live independently.  These data are used by Flight Offers Search as well for instance. Every year we do a Capture (copy of Prod) and every 3/6 months a restore (reset of test with Capture data).
For the feed part (OAG and others), it simply does not exist. Test is isolated from the outside, that's why we do not get any update.

Reply:
Thanks a lot for the feedback!